src/www/status_graph.php

```diff
@@ -48,6 +50,24 @@
     }
 }
 
+//Create array of hostnames from DHCP
+foreach ($interfaces as $ifname => $ifarr) {
+	if (isset($config['dhcpd'][$ifname]['staticmap'])) {
```

Comment:
Hi Jeff, this could be simplified by adding an additional loop and reuse the the logic underneath, like:
```

foreach (array('dhcpd', 'dhcpdv6') as $dhcp) {
    foreach($config[$dhcp][$ifname]['staticmap'] as $entry) {
        ...
    }
}
```